jcommune-model/src/main/java/org/jtalks/jcommune/model/entity/ObjectsFactory.java

```diff
@@ -254,4 +256,32 @@ public static Post getPostWithComments() {
         return post;
     }
 
+    public static List<Topic>  getTopicsWithSpecChars(){
+        JCUser user = new JCUser("username", "email", "password");
+        List<Topic> topicsWithSpecChars = new ArrayList<>();
+        Post postWithSpecChars = new Post(user, "����\u000F���");
+        Topic topicWithSpecChars = new Topic(user, "");
+        topicWithSpecChars.addPost(postWithSpecChars);
+        topicWithSpecChars.setId(2L);
+        topicsWithSpecChars.add(topicWithSpecChars);
+
+        return topicsWithSpecChars;
+    }
+
+    public static Map<String, Object>  getNewsModel()  {
```

Comment:
Hm.. I was sure I wrote this.. There is no such thing in the system as NewsModel. Therefore people won't understand it. You need to return a list of topics, not a map of abstract things.